__tests__/app.test.js

```diff
@@ -62,7 +66,8 @@ describe ('GET /api/articles/:article_id',() => {
         expect(article).toHaveProperty("topic", expect.any(String));  
         expect(article).toHaveProperty("created_at");    
         expect(article).toHaveProperty("votes", expect.any(Number));   
-        expect(article).toHaveProperty("article_img_url", expect.any(String));   
+        expect(article).toHaveProperty("article_img_url", expect.any(String)); 
+        expect(article).toHaveProperty("comment_count", expect.any(Number));   
```

Comment:
This should be a new test, not a refactor of the previous one ❗ Please add 400 and 404 tests too 👍

Reply:
I have add a new test for it